packages/desktop-client/src/components/budget/rollover/TransferTooltip.tsx

```diff
@@ -69,7 +82,7 @@ export default function TransferTooltip({
         <InitialFocus>
           <Input
             value={amount}
-            onChange={e => setAmount(e.target.value)}
+            onChange={e => setAmount(e.target['value'])}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏out of curiosity - why did you change this here and in a bunch of other places?

Reply:
This is a way to tell Typescript that the property we are trying to get exists on the object. In this case `e.target` might or might not have `value` and we could either do `(e.target as HTMLInputElement).value` or "force" it via `e.target['value']`. Dunno if using `currentTarget` would [solve the problem properly](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12239) but that would have been a runtime change I didn't wanna add in this PR.